app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Admin Invoice Show</h1>
+
+<h3>Invoice Details</h3>
+<div id="invoice-<%= @invoice.id %>">
+<p>Invoice ID:<%= @invoice.id%><p>
+<p>Invoice Status:<%=@invoice.status%><p>
+<p>Invoice Creation Date:<%=@invoice.created_at.to_time.strftime(' %A, %B %e, %Y')%><p>
+<p>Invoice Customer:<%=@invoice.customer.first_name%> <%=@invoice.customer.last_name%><p>
```

Comment:
You could potentially create an instance method on the customer model called 'full name' so that you can do this in one method call. It's pretty nitpicky though and I don't think you'd find another use for it besides this scenario.